src/components/Utils/resources.js

```diff
@@ -235,18 +236,13 @@ export const SUFFIXES_SETTING = {
 
 export const ORDER_TEMPLATES = {
   ...BASE_RESOURCE,
-  path: CONFIG_API,
-  records: 'configs',
-  GET: {
-    params: {
-      query: `(module=${MODULE_ORDERS} and configName=${CONFIG_ORDER_TEMPLATES})`,
-    },
-  },
+  path: ORDER_TEMPLATES_API,
+  records: 'orderTemplates',
```

Comment:
`{ params: query: 'cql.allRecords=1 sortby  templateName' }` for default alphabetic order

Reply:
updated